src/main/java/teammates/ui/controller/InstructorCourseJoinConfirmationPageData.java

```diff
@@ -1,14 +1,35 @@
 package teammates.ui.controller;
 
 import teammates.common.datatransfer.AccountAttributes;
+import teammates.common.util.Const;
+import teammates.common.util.Sanitizer;
 
 public class InstructorCourseJoinConfirmationPageData extends PageData {
 
-    public String regkey;
-    public String institute;
+    private String regkey;
+    private String institute;
     
-    public InstructorCourseJoinConfirmationPageData(AccountAttributes account) {
+    public InstructorCourseJoinConfirmationPageData(AccountAttributes account, String regkey, String institute) {
         super(account);
+        this.regkey = regkey;
+        this.institute = institute;
+    }
+    
+    public String getRegkey() {
+        return regkey;
+    }
+    
+    public String getInstitue() {
+        return institute;
```

Comment:
Typo in the method name.
It's easiest to just generate them in an IDE rather than typing them out manually.